src/test/java/uk/gov/digital/ho/proving/income/audit/AuditClientTest.java

```diff
@@ -128,13 +137,103 @@ public void shouldRetrieveAuditHistory() {
 
         URI uri = captorUri.getValue();
         assertThat(uri).hasHost(SOME_HISTORY_ENDPOINT.replace("http://", ""));
-        assertThat(uri).hasQuery(String.format("toDate=%s&eventTypes=%s", LocalDate.now().toString(), eventTypes.toString()));
         assertThat(auditRecords).isEqualTo(results);
+        String[] queryStringComponents = uri.getQuery().split("&");
+        assertThat(queryStringComponents).containsExactlyInAnyOrder(
+            "eventTypes=" + eventTypes,
+            "page=0",
+            "size=" + HISTORY_PAGE_SIZE,
+            "toDate=" + LocalDate.now().toString()
+        );
+    }
+
+    @Test
+    public void shouldRetrieveAuditHistory_returnsAllRecords() throws IOException {
```

Comment:
This test should be renamed to getAuditHistory_returnsAllRecords as getAuditHistory is the thing being tested. Following tests should be the same